{
  "info": {
    "author": "Mohd. Shafikur Rahman",
    "author_email": "shafikshaon@gmail.com",
    "bugtrack_url": null,
    "classifiers": [
      "License :: OSI Approved :: MIT License",
      "Operating System :: OS Independent",
      "Programming Language :: Python :: 2.7",
      "Programming Language :: Python :: 3"
    ],
    "description": "",
    "description_content_type": "",
    "docs_url": null,
    "download_url": "",
    "downloads": {
      "last_day": -1,
      "last_month": -1,
      "last_week": -1
    },
    "home_page": "https://github.com/shafikshaon/django-debug-helper",
    "keywords": "",
    "license": "",
    "maintainer": "",
    "maintainer_email": "",
    "name": "django-debug-helper",
    "package_url": "https://pypi.org/project/django-debug-helper/",
    "platform": "",
    "project_url": "https://pypi.org/project/django-debug-helper/",
    "project_urls": {
      "Homepage": "https://github.com/shafikshaon/django-debug-helper"
    },
    "release_url": "https://pypi.org/project/django-debug-helper/1.0.0/",
    "requires_dist": null,
    "requires_python": "",
    "summary": "Django debug error related exception search on Google/ Stack Overflow.",
    "version": "1.0.0",
    "yanked": false,
    "yanked_reason": null
  },
  "last_serial": 4471849,
  "urls": [
    {
      "comment_text": "",
      "digests": {
        "blake2b_256": "9bb0fdad64f128207798541d6fdceebe4a390cb6b3e0e78039fdc7500d4e9cb5",
        "md5": "636a58e250ec2d25985a72a7297aa436",
        "sha256": "a9dfbeb04ae71a80937df4841eb81465eb01c2091590aa65b1d1ad53c9b23524"
      },
      "downloads": -1,
      "filename": "django_debug_helper-1.0.0-py3-none-any.whl",
      "has_sig": false,
      "md5_digest": "636a58e250ec2d25985a72a7297aa436",
      "packagetype": "bdist_wheel",
      "python_version": "py3",
      "requires_python": null,
      "size": 2127,
      "upload_time": "2018-11-09T18:18:13",
      "upload_time_iso_8601": "2018-11-09T18:18:13.549250Z",
      "url": "https://files.pythonhosted.org/packages/9b/b0/fdad64f128207798541d6fdceebe4a390cb6b3e0e78039fdc7500d4e9cb5/django_debug_helper-1.0.0-py3-none-any.whl",
      "yanked": false,
      "yanked_reason": null
    },
    {
      "comment_text": "",
      "digests": {
        "blake2b_256": "ad84836614fe4284aa8f9fe7a48c5e7fdfc1119d20c00ada4121a164b4a87d62",
        "md5": "339b10393f46caebd935e49d9e736e06",
        "sha256": "8bacabede240a85785d5bd3556fbd12c1f378059304a795bd382110b16fb56e9"
      },
      "downloads": -1,
      "filename": "django-debug-helper-1.0.0.tar.gz",
      "has_sig": false,
      "md5_digest": "339b10393f46caebd935e49d9e736e06",
      "packagetype": "sdist",
      "python_version": "source",
      "requires_python": null,
      "size": 942,
      "upload_time": "2018-11-09T18:18:15",
      "upload_time_iso_8601": "2018-11-09T18:18:15.648887Z",
      "url": "https://files.pythonhosted.org/packages/ad/84/836614fe4284aa8f9fe7a48c5e7fdfc1119d20c00ada4121a164b4a87d62/django-debug-helper-1.0.0.tar.gz",
      "yanked": false,
      "yanked_reason": null
    }
  ],
  "vulnerabilities": []
}